Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-UserAdded_to_SecurityAdmin.yaml

```diff
@@ -1,6 +1,9 @@
 id: 363ea6d1-b30d-4a44-b56a-63c3c8a99621
 name: User added to SQL Server SecurityAdmin Group
 description: |
+    This hunting query identifies user added in the SecurityAdmin group of SQL Server
```

Comment:
"This hunting query identifies user added in the SecurityAdmin group of SQL Server. It relies on the SQLEvent KQL Parser function."